### PR TITLE
chore(curriculum): remove jquery from Legacy Basic CSS Challenges

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This removes jQuery from the Basic CSS Challenges and using modern ChaiJS assertions. In addition a test involving `style` was updated that was broken.

There is a default style element on the preview pages now  that has a class of `fcc-hide-header`. Tests to see if the camper has created a proper style element have been broken because it will always grab the `style.fcc-hide-header` one first. 

Edit: Looks like there was already a solution I didn't see?

### CODE REVIEW

**Comment by moT01** [comment]
I am all for removing jQuery in tests - but is there any reason for it here? Like is the long-term goal to remove it completely, so we don't need it as a dependency? I think we would really like to stop maintaining these older challenges.

**Comment by a2937** [comment]
My end goal was to completely remove from the curriculum so it can be safely removed from the codebase.

File: curriculum/challenges/english/01-responsive-web-design/basic-css/use-rgb-to-mix-colors.md [hunk 1]
```diff
@@ -22,65 +22,57 @@ Replace the hex codes in our `style` element with their correct RGB values.
...
 
 You should use `rgb` for the color blue.
 
 ```js
-assert(
-  code.match(
-    /\.blue-text\s*{\s*color\s*:\s*rgb\(\s*0\s*,\s*0\s*,\s*255\s*\)\s*;?\s*}/gi
-  )
-);
+assert.match(code,/\.blue-text\s*{\s*color\s*:\s*rgb\(\s*0\s*,\s*0\s*,\s*255\s*\)\s*;?\s*}/gi);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/01-responsive-web-design/basic-css/use-rgb-to-mix-colors.md:61]:
```suggestion
assert.match(__helpers.removeCssComments(code), /\.blue-text\s*{\s*color\s*:\s*rgb\(\s*0\s*,\s*0\s*,\s*255\s*\)\s*;?\s*}/gi);
```

File: curriculum/challenges/english/01-responsive-web-design/basic-css/use-rgb-to-mix-colors.md [hunk 2]
```diff
@@ -22,65 +22,57 @@ Replace the hex codes in our `style` element with their correct RGB values.
...
 
 You should use `rgb` for the color sienna.
 
 ```js
-assert(
-  code.match(
-    /\.sienna-text\s*{\s*color\s*:\s*rgb\(\s*160\s*,\s*82\s*,\s*45\s*\)\s*;?\s*}/gi
-  )
-);
+assert.match(code,/\.sienna-text\s*{\s*color\s*:\s*rgb\(\s*160\s*,\s*82\s*,\s*45\s*\)\s*;?\s*}/gi);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/01-responsive-web-design/basic-css/use-rgb-to-mix-colors.md:75]:
```suggestion
assert.match(__helpers.removeCssComments(code), /\.sienna-text\s*{\s*color\s*:\s*rgb\(\s*160\s*,\s*82\s*,\s*45\s*\)\s*;?\s*}/gi);
```

File: curriculum/challenges/english/01-responsive-web-design/basic-css/use-rgb-values-to-color-elements.md [hunk 3]
```diff
@@ -44,13 +44,15 @@ Let's replace the hex code in our `body` element's background color with the RGB
 Your `body` element should have a black background.
 
 ```js
-assert($('body').css('background-color') === 'rgb(0, 0, 0)');
+const body = document.querySelector('body');
+const backgroundColor = window.getComputedStyle(body)['background-color']; 
+assert.strictEqual(backgroundColor,'rgb(0, 0, 0)');
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/01-responsive-web-design/basic-css/use-rgb-values-to-color-elements.md:49]:
```suggestion
assert.strictEqual(backgroundColor, 'rgb(0, 0, 0)');
```

File: curriculum/challenges/english/01-responsive-web-design/basic-css/use-rgb-values-to-color-elements.md [hunk 4]
```diff
@@ -44,13 +44,15 @@ Let's replace the hex code in our `body` element's background color with the RGB
...
 ```
 
 You should use `rgb` to give your `body` element a background of black.
 
 ```js
-assert(code.match(/rgb\s*\(\s*0\s*,\s*0\s*,\s*0\s*\)/gi));
+assert.match(code,/rgb\s*\(\s*0\s*,\s*0\s*,\s*0\s*\)/gi);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/01-responsive-web-design/basic-css/use-rgb-values-to-color-elements.md:55]:
```suggestion
assert.match(__helpers.removeCssComments(code), /rgb\s*\(\s*0\s*,\s*0\s*,\s*0\s*\)/gi);
```

**Review by naomi-lgbt — APPROVED** [review]

**Comment by ilenia-magoni** [comment]
@Naomi can you confirm it is `removeHtmlComments` and not `removeHTMLComments`? if so I will change the documentation

**Review by majestic-owl448 — APPROVED** [review]

**Pull request merged**